src/experiment.py

```diff
@@ -0,0 +1,188 @@
+from optimisers import Problem, Optimiser
+from sensor_group import SensorSuite
+from fields import Field
+import numpy as np
+
+
+
+
+class Experiment():
+    """
+    Manages the optimisers, true field and sensor suite.
+    3 main functions.
+    1. Initialisation to define experiment parameters.
+    2. Planning to prepare for optimisation.
+    3. Design to optimise the experiment.
+    """
+    def __init__(self, true_field :Field, comparison_pos :np.ndarray[float], optimiser :Optimiser) -> None:
+        """
+        Args:
+            true_field (Field): the field we got from the simulation - that we expect to measure in an experiment.
```

Comment:
```suggestion
            true_field (Field): the simulated field which acts as the ground truth against which to compare the predicted field.
```
Note that in the 3D case we don't expect to measure the full field by experiment, since the sensors can only detect the surface.